apps/web/src/services/asset-inventory/collector/collector-history/CollectorHistoryPage.vue

```diff
@@ -53,7 +53,7 @@
                         :text-color="statusTextColorFormatter(value)"
                         :icon="statusIconFormatter(value)"
                         :icon-color="statusIconColorFormatter(value)"
-                        :icon-animation="[JOB_STATE.IN_PROGRESS, JOB_STATE.CREATED].includes(value) ? 'spin' : undefined"
+                        :icon-animation="[JOB_STATE.IN_PROGRESS].includes(value) ? 'spin' : undefined"
```

Comment:
nitpick: If there won't be any additions to the array later on, this would be better!

```
:icon-animation="value === JOB_STATE.IN_PROGRESS ? 'spin' : undefined"
```